### ISSUE TITLE
Connections disappear

### ISSUE THREAD

**Issue by KupueIIIKo** [ashkulz/NppFTP#238]
Good day! Sorry for English, I use a translator. I work with your library for a long time! She is comfortable! Thank! But now there is a problem for several years! When you add a new connection and specify the login in the Russian layout, sorting occurs. When you turn off notepad ++ and turn it on again, the recording with Russian login and everything disappears after it! Best regards, Sergei!

**Comment by chcg** [comment]
@KupueIIIKo Which version of NppFTP do you use?

I think I could still reproduce the issue with the latest version 0.28.0 with e.g. Полгода as login. After closing the profile dialog and opening again all the characters are changed to just ???????. Seems like the login is not capable to handle UNICODE chars correctly.
Is this what your report is about?

**Comment by KupueIIIKo** [comment]
Yes you can play at 0.28. Yesterday checked. Even on different versions on different computers on the windows 10 operating system. After closing the notepadd ++ in general! That is, let's say the plugin should restart.

**Comment by polo19** [comment]
ya ya drug, toje samoe
